src/components/molecule.js

```diff
@@ -75,6 +75,13 @@ class Molecule extends Component {
     }
   }
 
+  formatLink = (props) => {
+    if (!isNil(props.name)) {
+      return <a href src={props.wikipediaUrl}>{props.name}</a>
+    }
+    return <a href src={props.wikipediaUrl}>Wikipedia Page</a>
+  }
+
```

Comment:
Where is this used?

Reply:
In the `Molecule Properties` card on the molecule page the Wikipedia link is displayed if it is provided. The link text should be the molecule name, but if for some reason no common name has been provided the text will default to being 'Wikipedia Page'.

Reply:
Sorry, misunderstood the question. The function was not being called, but should have been. This has been fixed now.